publications-2021.bib

```diff
@@ -668,4 +668,99 @@ @article{maier21c
   pages = {20210609},
 }
 
+
+@unpublished{160_PREPRINT,
```

Comment:
Is it still a preprint? I found this article here: https://link.springer.com/chapter/10.1007/978-3-030-87312-7_24

Could you add the doi and update the journal etc.?

Reply:
Yes indeed, that is already published by now. I updated the entry